package.json

```diff
@@ -3,11 +3,12 @@
     "version": "1.0.4",
     "private": true,
     "scripts": {
-        "prestart": "yarn run buildplugin && yarn link docusaurus-plugin-navdata",
-        "buildplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && yarn link",
+        "prestart": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner",
+        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata  && yarn install && yarn link",
```

Comment:
Are you sure linking works properly? Because I can see warning that linking has no effect:

```
yarn buildnavplugin
yarn run v1.22.19
$ cd ./src/plugins/docusaurus-plugin-navdata  && yarn install && yarn link
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
$ tsdx build
@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
✓ Creating entry file 1.4 secs
✓ Building modules 2.2 secs
warning There's already a package called "docusaurus-plugin-navdata" registered. This command has had no effect. If this command was run in another folder with the same name, the other folder is still linked. Please run yarn unlink in the other folder if you want to register this folder.
Done in 5.24s.
```

I am worried that having two project versions on the same machine might result in strange errors.

Reply:
The linking it's only needed to be made just one time, but we're leaving the script there just in case something change in the future, and the linking needs to be remade.